src/elements/common/_animations.scss

```diff
@@ -0,0 +1,27 @@
+@keyframes fade-in {
+    0% {
+        opacity: 0;
+    }
+
+    100% {
+        opacity: 1;
+    }
+}
+
+@mixin fade-in($fade-in-time) {
+    animation: fade-in ease-in $fade-in-time;
+}
+
+@mixin loading-pulse($start-color, $end-color, $fade-in-time) {
```

Comment:
@ConradJChan we should be able to reuse this for the Content Picker loading state.